* use prefix bash
* use /var/tmp iso /tmp for Solaris' sake

--- xmlto.in
+++ xmlto.in
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!@BASH@
 # 
 # @PACKAGE@ - apply an XSL stylesheet to an XML document
 # Copyright (C) 2001, 2002, 2003  Tim Waugh <twaugh@redhat.com>
@@ -105,7 +105,7 @@
 
   if [ -n "$papertype" ]
   then
-    papersizemod=$(${MKTEMP} ${TMPDIR:-/tmp}/@PACKAGE@-xsl.XXXXXX)
+    papersizemod=$(${MKTEMP} ${TMPDIR:-/var/tmp}/@PACKAGE@-xsl.XXXXXX)
     CLEANFILES[$CLEANFILE_COUNT]="$papersizemod"
     CLEANFILE_COUNT=$(($CLEANFILE_COUNT + 1))
     cat << EOF > "$papersizemod"
@@ -287,7 +287,7 @@
 then
   REAL_STYLESHEET="$STYLESHEET"
   [ "$VERBOSE" -ge 1 ] && echo >&2 "Real stylesheet: ${REAL_STYLESHEET}"
-  STYLESHEET="$(${MKTEMP} ${TMPDIR:-/tmp}/@PACKAGE@-xsl.XXXXXX)" || exit 1
+  STYLESHEET="$(${MKTEMP} ${TMPDIR:-/var/tmp}/@PACKAGE@-xsl.XXXXXX)" || exit 1
   CLEANFILES[$CLEANFILE_COUNT]="$STYLESHEET"
   CLEANFILE_COUNT=$(($CLEANFILE_COUNT + 1))
   cat << EOF > "$STYLESHEET"
@@ -311,7 +311,7 @@
 EOF
 fi
 
-XSLT_PROCESSED_DIR="$(${MKTEMP} -d ${TMPDIR:-/tmp}/@PACKAGE@.XXXXXX)" || exit 1
+XSLT_PROCESSED_DIR="$(${MKTEMP} -d ${TMPDIR:-/var//tmp}/@PACKAGE@.XXXXXX)" || exit 1
 CLEANFILES[$CLEANFILE_COUNT]="$XSLT_PROCESSED_DIR"
 CLEANFILE_COUNT=$(($CLEANFILE_COUNT + 1))
 cd "$XSLT_PROCESSED_DIR"
